python/ray/autoscaler/aws/tests/aws_compute.yaml

```diff
@@ -1,4 +1,4 @@
-cloud_id: cld_17WvYIBBkdgLwEUNcLeRAE
+cloud_id: {{env["ANYSCALE_CLOUD_ID"]}}
```

Comment:
qq, where is this specified? Or is that up to the test runner to specify this?

Reply:
totally, it's define in an .env file https://github.com/ray-project/ray/blob/master/release/ray_release/environments/staging_v2.env#L3